### DISCUSSION TITLE
Some bad news, and some good

### DISCUSSION THREAD

**Discussion by fjwillemsen** [fjwillemsen/NativeOverleaf#38] [Announcements]
First, the bad news. As many of you are aware, recent changes to Overleaf have made it very hard to keep this project in good condition. For example, the settings panel of Native Overleaf was broken by changes on Overleaf's end. I have tried my best to work around these issues, but this is not sustainable. This means that Native Overleaf is not able to provide the stability needed to use it in daily work, and I will no longer be working towards fixing the individual issues. 

The good news: I am very grateful for the overwhelmingly positive response Native Overleaf has had over the past two years and would like to create something even better. I believe it is necessary to start from the ground up to create a completely new solution that will not have these issues, something worthy of serving the academic community. I am currently working on this, and hope to share this with all of you soon.

**Comment by nkapila6** [comment]
Thanks for all your efforts, Floris. 

Waiting for the next iteration of NativeOverleaf!

**Comment by quarkquartet** [comment]
Very glad and touched to hear this. Thank you for your hard work and looking forward to the new app!

**Comment by RhvdG** [comment]
Thank you for your continued efforts!

**Comment by Frzgunr1** [comment]
thx！
